microprot/scripts/process_input.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import click
+import os
+from microprot.scripts import process_fasta
+
+
+@click.command()
+@click.option('--infile', '-i', required=True,
+              type=click.Path(resolve_path=True, readable=True, exists=True),
+              help='Input protein sequence file in FASTA format.')
+@click.option('--outfile', '-o', required=False,
+              type=click.Path(resolve_path=True, readable=True, exists=False),
+              help='Output protein sequence file in FASTA format.')
+@click.option('--sort', '-s', required=False, is_flag=True,
+              help='Sort sequences by length')
+@click.option('--min_len', '-m', required=False, default=1,
+              help='Minimum sequence length to be included in output.')
+@click.option('--max_len', '-x', required=False, default=10000,
+              help='Maximum sequence length to be included in output.')
+def _process_fasta_input(infile, outfile, sort, min_len, max_len):
+    fp = infile
+    fp_name = os.path.splitext(fp)[0]
+
+    fasta = process_fasta.extract_sequences(fp)
+    fasta = np.array(fasta)
+
+    suffix = []
+
+    if sort is True:
```

Comment:
The variable name `sort` looks so close to reserved Python keywords. Maybe do `sort_by_length`?